webpack/assets/javascripts/react_app/redux/reducers/index.js

```diff
@@ -0,0 +1,6 @@
+import { combineReducers } from 'redux';
+import statistics from './statistics';
+
+export default combineReducers({
```

Comment:
is this a global file for all reduces? wont it grow quickly for every component?